components/x11/xf86-video-ati/Makefile

```diff
@@ -33,12 +34,14 @@ CONFIGURE_DEFAULT_DIRS = no
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
-COMPONENT_PREP_ACTION =        ( cd $(@D) && \
-                                        aclocal -I. && \
-                                        autoheader && \
-                                        libtoolize --automake --copy --force && \
-                                        automake -a -f -c &&\
-                                        autoconf )
+COMPONENT_PREP_ACTION = \
+ ( cd $(@D) && \
+    aclocal -I. && \
+    autoheader && \
+    libtoolize --automake --copy --force && \
+    automake -a -f -c && \
+    autoconf && \
+    cp $(COMPONENT_DIR)/srcs/i2c/*.h $(COMPONENT_DIR)/srcs/i2c/*.c src/ )
```

Comment:
The i2c code is not provided by Xorg since 2014 so it should be provided and compiled in xorg-video-ati.
If you have a better idea how to place the missing source in the tree I can change it but I do not understand your question about $(BUILD_DIR).